src/pal/src/thread/thread.cpp

```diff
@@ -2633,6 +2633,8 @@ CPalThread::GetStackBase()
     status = pthread_attr_get_np(thread, &attr);
 #elif HAVE_PTHREAD_GETATTR_NP
     status = pthread_getattr_np(thread, &attr);
+#elif __ANDROID__
```

Comment:
We should rather fix the detection of HAVE_PTHREAD_GETATTR_NP to work on Android too.